### ISSUE TITLE
HTML errors with language toggle

### ISSUE THREAD

**Issue by piet0024** [gctools-outilsgc/gcconnex#928]
```
<span class="active"><lan=en>English</span>
<a class="not_active" href="javascript:form_submit('French')"><lan=fr>Fran&ccedil;ais<lan=en></a>
```

Should be `<span lang="en"></span>`

This is also an issue for collab

@CoolColin
